src/OrbitClientModel/include/OrbitClientModel/CaptureData.h

```diff
@@ -239,7 +243,7 @@ class CaptureData {
 
   std::chrono::system_clock::time_point capture_start_time_ = std::chrono::system_clock::now();
 
-  UserDefinedCaptureData user_defined_capture_data_;
+  absl::flat_hash_set<uint64_t> frame_track_function_ids_;
```

Comment:
As said above, I am really unsure wether we want this change or not. UserDefinedCaptureData can change even after a capture was taken. This was encapsulated by having this separate class. Also IIRC we planned to add additional stuff there.

Reply:
I think we want this change. Note that we still use UserDefinedCaptureData and function set in datamanager, but for capturedata this is not needed because here we have ids for these functions.